Microsoft.Toolkit.Uwp.UI.Animations/Options/RepeatOption.cs

```diff
@@ -0,0 +1,114 @@
+        /// </summary>
+        public static RepeatOption One => new(1);
+
+        /// <summary>
+        /// Gets a <see cref="RepeatOption"/> value indicating an animation that repeats forever.
+        /// </summary>
+        public static RepeatOption Forever => new(-1);
+
+        /// <summary>
+        /// Creates a <see cref="RepeatOption"/> value with the specified number of iterations.
+        /// </summary>
+        /// <param name="count">The number of iterations for the animation.</param>
+        /// <returns>A <see cref="RepeatOption"/> value with the specified number of iterations.</returns>
+        /// <exception cref="ArgumentOutOfRangeException">Thrown when <paramref name="count"/> is negative.</exception>
+        [Pure]
+        public static RepeatOption Count(int count)
+        {
+            Guard.IsGreaterThanOrEqualTo(count, 0, nameof(count));
+
+            return new(count);
+        }
+
+        /// <summary>
+        /// Parses a <see cref="RepeatOption"/> value from a <see cref="string"/>.
+        /// The allowed values are either non-negative integers, or "Forever".
+        /// </summary>
+        /// <param name="text">The input text to parse.</param>
+        /// <returns>The parsed <see cref="RepeatOption"/> value.</returns>
+        [Pure]
+        public static RepeatOption Parse(string text)
+        {
+            if (int.TryParse(text, out int count))
+            {
+                return Count(count);
+            }
+
+            if (text.AsSpan().Trim().SequenceEqual("Forever".AsSpan()))
+            {
+                return Forever;
+            }
+
+            return ThrowHelper.ThrowArgumentException<RepeatOption>("Invalid input text");
+        }
+
+        /// <summary>
+        /// Gets a <see cref="RepeatBehavior"/> value corresponding to the current <see cref="RepeatOption"/> value.
+        /// </summary>
+        /// <returns>A <see cref="RepeatBehavior"/> value matching the current <see cref="RepeatOption"/> value.</returns>
+        [Pure]
+        public RepeatBehavior ToRepeatBehavior()
+        {
+            if (this.value < 0)
+            {
+                return RepeatBehavior.Forever;
+            }
+
+            return new(this.value);
+        }
+
+        /// <summary>
+        /// Gets the <see cref="AnimationIterationBehavior"/> and count values matching the current <see cref="RepeatOption"/> value.
+        /// If the current value represents an infinitely repeating animation, the returned count will be set to 1.
+        /// </summary>
+        /// <returns>The <see cref="AnimationIterationBehavior"/> and count values matching the current <see cref="RepeatOption"/> value.</returns>
+        [Pure]
+        public (AnimationIterationBehavior Behavior, int Count) ToBehaviorAndCount()
+        {
+            if (this.value < 0)
+            {
+                return (AnimationIterationBehavior.Forever, 1);
```

Comment:
Is it weird that Count is 1 here or is this pretty much ignored?

Reply:
I'm just returning `1` here as it's the default value, but when setting `AnimationIterationBehavior.Forever` in an animation it'll be ignored anyway. It's just for simplicity to avoid returning actually invalid values like eg. `0` or `-1` (I've documented this in the XML docs). The idea is that whatever `RepeatOption` value you have, calling this method and directly assigning the returned values to an animation should result in the correct behavior without crashes or other checks needed 🙂